en/telemetry/holybro_microhard_p900_radio.md

```diff
@@ -88,11 +74,20 @@ You can purchase a [Holybro UART to USB Converter](https://shop.holybro.com/gps-
 *Pico Config* will automatically detect and connect to the configuration port.
 Adjust the settings so that the baud rate matches PX4 (and the ground station).
 
-Once the radios and PX4 are configured to use the same baud rate you can connect *QGroundControl* to the vehicle via the radio.
-*QGroundControl* will not automatically detect the radio for anything other than the default baud rate, so first create a new "serial connection" in [Application Settings > Comm Links](https://docs.qgroundcontrol.com/master/en/SettingsView/SettingsView.html) (set the same baud rate as used by PX4/radios).
-
 Holding the **Config** button while powering-up the radio will boot the unit into COMMAND mode: the default serial interface will be active and temporarily set to operate at its default serial settings of 9600/8/N/1.
 
+Note that it is also possible to configure the radios using AT commands through the data port.
```

Comment:
@vincentpoont2 Note, that I have updated the docs to point to the common configuration in the parent page. The general bit that is related to connecting up a particular device remains of course.

The reason this note is here is because originally you said we can configure the device in two ways, and AT commands through the data port is one of them. However there are no instructions on that and the general setup that is explained is the one to do with using Picoconfig. So I have moved this to be a note at the end, and you can provide extra information on how to use the data port if you think it is useful.